packages/app/src/connecting/components/connecting.component.tsx

```diff
@@ -47,7 +48,9 @@ const Connecting: FunctionComponent<{
   updateAllIndexes,
   forceOsUpdateFailed,
   checkingForOsForceUpdate,
+  passcodeModalCloseable,
 }) => {
+  console.log("mw_", "passcodeModalCloseable", passcodeModalCloseable)
```

Comment:
log 😉

Reply:
Uf, good that this console.log is without any swear word :D 

Changed in https://github.com/mudita/mudita-center/pull/1103/commits/fcd64937f3f30b081e8d6cae273943e75bb0c3f5